discovery-provider/src/tasks/celery_app.py

```diff
@@ -4,3 +4,9 @@
 # Create the celery application
 # Invoked by worker.py
 celery = Celery(__name__)
+
+# Celery removes all configured loggers. This setting prevents
+# Celery from overriding the configured logger set in create_celery() 
+celery.conf.update({
+    'worker_hijack_root_logger': False
```

Comment:
wow this explains a lot! so w/this set as false, can we configure our logger w the actual loglevel_celery value?

Reply:
yup!! :D except log level for some reason isn't accepted by celery unless it's via command line